src/modules/product-category/entities/product-category.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { AbstractBaseEntity } from "../../../entities/base.entity";
+import { Product } from "../../products/entities/product.entity";
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+
+// @Entity()
+// export class ProductCategory extends AbstractBaseEntity {
+// @Column({type:'text', unique: true})
+// name:string
+// @Column({type:'text', nullable: true})
+// description: string
+// @OneToMany(()=> Product, products => products.category)
+// product:Product
+// }
+
+@Entity('categories')
+export class ProductCategory extends AbstractBaseEntity{
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
```

Comment:
You already have this when you `extend` AbstractBaseEntity. It can be removed

Reply:
Done